app/domain/logs.rb

```diff
@@ -555,6 +555,41 @@ module AuthnJwt
         code: "CONJ00109D"
       )
 
+      FetchingIdentityPath = ::Util::TrackableLogMessageClass.new(
+        msg: "Fetching identity path...",
+        code: "CONJ00110D"
+      )
+
+      FetchedIdentityPath = ::Util::TrackableLogMessageClass.new(
+        msg: "Successfully fetched identity path '{0-identity-path}'",
+        code: "CONJ00111I"
+      )
+
+      FetchingIdentityByInterface = ::Util::TrackableLogMessageClass.new(
+        msg: "Fetching identity by interface: '{0-interface-name}'",
```

Comment:
```suggestion
        msg: "Fetching JWT identity by interface: '{0-interface-name}'...",
```
what do you mean by "interface"? Can you provide an example?

Reply:
Technical term which indicate on a specific flow in the code

```
CONJ00058D Selecting identity provider interface...
CONJ00059I Selected identity provider interface: 'token-identity-provider'
CONJ00114D Fetching identity by interface: 'token-identity-provider'
```

We currently have to interfaces in the code: 'token-identity-provider' 'url-identity-provider' which indicating from where we will fetch the identity of the token.

In the future we may have more interface implementations